recipes/cli11/all/test_package/conanfile.py

```diff
@@ -1,17 +1,27 @@
+from conan import ConanFile
+from conan.tools.build import can_run
+from conan.tools.cmake import cmake_layout, CMake
 import os
-from conans import ConanFile, CMake, tools
 
 
+# It will become the standard on Conan 2.x
```

Comment:
```suggestion
```

Please, avoid comments from templates, it's documented to be removed: https://github.com/conan-io/conan-center-index/blob/master/docs/package_templates/cmake_package/all/conanfile.py#L15